tests/bluetooth/controller/ctrl_conn_update/src/main.c

```diff
@@ -2574,81 +2574,81 @@ void test_main(void)
-			 ztest_unit_test_setup_teardown(test_conn_update_mas_rem_collision, setup,
+			 ztest_unit_test_setup_teardown(test_conn_update_cen_rem_collision, setup,
 							unit_test_noop));
 
 	ztest_test_suite(
-		sla_loc,
-		ztest_unit_test_setup_teardown(test_conn_update_sla_loc_accept, setup,
+		per_loc,
+		ztest_unit_test_setup_teardown(test_conn_update_per_loc_accept, setup,
 					       unit_test_noop),
-		ztest_unit_test_setup_teardown(test_conn_update_sla_loc_reject, setup,
+		ztest_unit_test_setup_teardown(test_conn_update_per_loc_reject, setup,
 					       unit_test_noop),
-		ztest_unit_test_setup_teardown(test_conn_update_sla_loc_unsupp_feat_wo_feat_exch,
+		ztest_unit_test_setup_teardown(test_conn_update_per_loc_unsupp_feat_wo_feat_exch,
 					       setup, unit_test_noop),
-		ztest_unit_test_setup_teardown(test_conn_update_sla_loc_unsupp_feat_w_feat_exch,
+		ztest_unit_test_setup_teardown(test_conn_update_per_loc_unsupp_feat_w_feat_exch,
 					       setup, unit_test_noop),
-		ztest_unit_test_setup_teardown(test_conn_update_sla_loc_collision, setup,
+		ztest_unit_test_setup_teardown(test_conn_update_per_loc_collision, setup,
 					       unit_test_noop));
 
-	ztest_test_suite(sla_rem,
-			 ztest_unit_test_setup_teardown(test_conn_update_sla_rem_accept, setup,
+	ztest_test_suite(per_rem,
+			 ztest_unit_test_setup_teardown(test_conn_update_per_rem_accept, setup,
 							unit_test_noop),
-			 ztest_unit_test_setup_teardown(test_conn_update_sla_rem_reject, setup,
+			 ztest_unit_test_setup_teardown(test_conn_update_per_rem_reject, setup,
 							unit_test_noop),
-			 ztest_unit_test_setup_teardown(test_conn_update_sla_rem_unsupp_feat, setup,
+			 ztest_unit_test_setup_teardown(test_conn_update_per_rem_unsupp_feat, setup,
 							unit_test_noop),
-			 ztest_unit_test_setup_teardown(test_conn_update_sla_rem_collision, setup,
+			 ztest_unit_test_setup_teardown(test_conn_update_per_rem_collision, setup,
 							unit_test_noop));
 
-	ztest_run_test_suite(mas_loc);
-	ztest_run_test_suite(mas_rem);
-	ztest_run_test_suite(sla_loc);
-	ztest_run_test_suite(sla_rem);
+	ztest_run_test_suite(cen_loc);
+	ztest_run_test_suite(cen_rem);
+	ztest_run_test_suite(per_loc);
+	ztest_run_test_suite(per_rem);
 
 #else /* !CONFIG_BT_CTLR_CONN_PARAM_REQ */
 
-	ztest_test_suite(mas_loc_no_param_req, ztest_unit_test_setup_teardown(
-				 test_conn_update_mas_loc_accept_no_param_req,
+	ztest_test_suite(cen_loc_no_param_req, ztest_unit_test_setup_teardown(
+				 test_conn_update_cen_loc_accept_no_param_req,
 				 setup, unit_test_noop));
 
-	ztest_test_suite(mas_rem_no_param_req, ztest_unit_test_setup_teardown(
-				 test_conn_update_mas_rem_accept_no_param_req,
+	ztest_test_suite(cen_rem_no_param_req, ztest_unit_test_setup_teardown(
+				 test_conn_update_cen_rem_accept_no_param_req,
 				 setup, unit_test_noop));
 
 	ztest_test_suite(
-		sla_loc_no_param_req,
-		ztest_unit_test_setup_teardown(test_conn_update_sla_loc_disallowed_no_param_req,
+		per_loc_no_param_req,
```

Comment:
Personally I think the `per` abbreviation should be avoided to avoid clashes with periodic advertising which is also often abbreviated to `per`.

Reply:
per is now replaced with periph, cen with central